src/AppBundle/Tiqr/TiqrFactory.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Copyright 2017 SURFnet B.V.
```

Comment:
Trivial: but all copyright statements are dated 2017, these files have been added in 2018 :)